HARK/ConsumptionSaving/tests/test_ConsPortfolioFrameModel.py

```diff
@@ -36,15 +36,19 @@ def test_simOnePeriod(self):
         self.pcct.track_vars += ["aNrm"]
         self.pcct.initialize_sim()
 
-        self.assertFalse(np.any(self.pcct.shocks["Adjust"]))
+        # simulation test -- seed/generator specific
```

Comment:
Is agent.pcct an analogue of agent.history?

Reply:
Here, `self.pcct` refers to the `AgentType` object.
This is testing the stored values of that object's `shocks` dictionary after some number of simulated steps.